Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py

```diff
@@ -160,35 +160,39 @@ def get_taxii(self, args: Dict[str, Any], is_first_fetch: bool = False):
                 raise ValueError("Last fetch time retrieval failed.")
 
             for data in self.fetch(args.get('begin'), args.get('end'), args.get('collection')):
-                skip = False
-                response = self.parse_to_json(data)
-
-                if response.get('indicators') or False:
-                    content = response.get('indicators')
-                elif response.get('ttps') or False:
-                    content = response.get('ttps').get('ttps')
-                else:
+                try:
+                    skip = False
+                    response = self.parse_to_json(data)
+
+                    if response.get('indicators') or False:
+                        content = response.get('indicators')
+                    elif response.get('ttps') or False:
+                        content = response.get('ttps').get('ttps')
+                    else:
+                        continue
+
+                    for eachone in content:
+                        if eachone.get('confidence'):
+                            current_timestamp = parser.parse(
+                                eachone['confidence']['timestamp']).replace(tzinfo=pytz.UTC).strftime(DATETIME_FORMAT)
+                            if is_first_fetch or datetime.fromisoformat(current_timestamp) > datetime.fromisoformat(save_fetch_time):
+                                save_fetch_time = current_timestamp
+                            else:
+                                skip = True
+
+                    if not skip:
+                        taxii_data.append(response)
+
+                        count += 1
+                        if count == args.get('limit'):
+                            break
+                except Exception as e:
+                    demisto.debug("Error with formatting feeds, exception:{}".format(e))
                     continue
 
-                for eachone in content:
-                    if eachone.get('confidence'):
-                        current_timestamp = parser.parse(
-                            eachone['confidence']['timestamp']).replace(tzinfo=pytz.UTC).strftime(DATETIME_FORMAT)
-                        if is_first_fetch or datetime.fromisoformat(current_timestamp) > datetime.fromisoformat(save_fetch_time):
-                            save_fetch_time = current_timestamp
-                        else:
-                            skip = True
-
-                if not skip:
-                    taxii_data.append(response)
-
-                    count += 1
-                    if count == args.get('limit'):
-                        break
-
         except Exception as e:
-            demisto.error("Failed to fetch feed details, exception:{}".format(e))
-            raise e
+            demisto.debug("Failed to fetch feed details, exception:{}".format(e))
+            return taxii_data, save_fetch_time
```

Comment:
Actually looking at this, I think there may be an issue which caused the UTs to fail. When the feed fails to fetch, we are just returning an empty result set and logging that there was a failure. Is the goal to just retry on the next fetch? 

This test is one of the failing ones:
```python
def test_get_taxii_invalid(mocker, capfd):
    from CybleThreatIntel import Client
    client = Client(params)

    mock_response_1 = str(open("test_data/data_err.xml", "r").read())
    mocker.patch.object(client, 'fetch', return_value=[mock_response_1])
    with capfd.disabled():
        try:
            val, time = Client.get_taxii(client, args2)
        except Exception as e:
            error_val = e.args[0]

    assert "Last fetch time retrieval failed." in error_val
```
Because no exception is raised, the test itself fails. Maybe the thing to do is to allow the exception to be raised and then in the function which calls it, catch the exception and set the default values there.

Reply:
Hello @amshamah419,
Occasionally, there are unexpected response from server, which is fine, because the feeds can be fetched in next call. But raising an error is something which is undesirable. So I have removed this test now.

You may proceed with review and merging the PR.

Thanks.